stable_baselines/ppo2/ppo2.py

```diff
@@ -114,6 +114,8 @@ def setup_model(self):
                 n_batch_step = None
                 n_batch_train = None
                 if issubclass(self.policy, LstmPolicy):
+                    assert self.nminibatches <= self.n_envs, "For recurrent policies, nminibatches should be smaller "\
```

Comment:
this is not quite right, ```self.n_envs``` must be a multiple of ```self.nminibatches```

Reply:
I think there is another check after that
but for recurrent policies, this is required (see https://github.com/hill-a/stable-baselines/issues/18)

Reply:
here: https://github.com/hill-a/stable-baselines/blob/c58ad7973d9a5dc753968a137ceea13f679df3a4/stable_baselines/ppo2/ppo2.py#L291

Reply:
I think i see your point... will test that